.github/workflows/release.yml

```diff
@@ -15,7 +14,12 @@ on:
       - '*'
     paths:
       - sed/**/*
+      - .github/workflows/release.yml
+    paths-ignore:
       - pyproject.toml
```

Comment:
https://github.com/OpenCOMPES/sed/actions/runs/6942391340
says this is not valid syntax

Reply:
Right. I just put it for extra certainty but eitherways the workflow won't trigger since it only ever changes the toml file and that's not in the paths.